src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -798,17 +794,16 @@ public void scrollToPosition(String selector) {
     jsActions.scrollToSpecificElement(driver.findElement(By.cssSelector(selector)));
     PageObjectLogging.log("scrollToSelector", "Scroll to the web selector " + selector, true);
   }
-  // This scroll has been created because ad is not displayed if we scroll quickly to the Footer ADEN-4359
-  public void scrollToBottomLeaderboard(){
-    scrollToFooter();
-    wait.forElementVisible(By.cssSelector(".editarea"));
-    scrollToFooter();
-  }
 
-  public void clickOnAdImage(String slotName){
-    adSelector(slotName);
-    adSelector(slotName).click();
-    PageObjectLogging.log("clickOnAdImage", slotName + " is clicked", true);
+  // This scroll has been created because ad is not displayed if we scroll quickly to the Footer ADEN-4359
+  public void scrollToSlot(String slotName) {
```

Comment:
We found today `triggerComments()` and used it here. But I'm still thinking we don't need `scrollToSlot()` especially it works only with two slots: bottom leaderboards (one slot per skin). Think about:
A) moving it into `TestAdsVUAP` class and re-use from them,
B) resigning from this method and using `scrollToElement()` with `triggerComments()` directly in the test's method.